app/Models/Signup.php

```diff
@@ -22,7 +22,7 @@ class Signup extends Model
      * @var array
      */
     public static $indexes = [
-        'campaign_id', 'campaign_run_id',
+        'campaign_id', 'campaign_run_id', 'updated_at',
```

Comment:
@katiecrane I'm not sure if I'm understanding correctly but this is already in the `signups` table! I just had to add it here so it would be able to be updated when anything else (post, reaction, review) is updated. LMK if that's not what you meant tho!